Fixtures/Schemes/AIS.xcscheme

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```

Comment:
Out of curiosity, what does `AIS` stand for?

Reply:
Automation Infrastructure Shared, which is an internal library for testing support. I wasn't sure if the name of a file loaded during a test case had much value, but I'd be happy to rename it to something a bit more useful. Perhaps LotsOfMissingInformation.xcscheme?